src/main/java/sawkiewicz/task1/CartItem.java

```diff
@@ -0,0 +1,34 @@
+package sawkiewicz.task1;
+
+public class CartItem {
+    int quantity;
```

Comment:
fields always should be `private`, rule of thumb is to keep class internals for it's own use

Reply:
done